### ISSUE TITLE
osm_id

### ISSUE THREAD

**Issue by klokan** [geometalab/open-streets.tm2source#4]
We should ensure osm_id are unique. Shall we do the same magic as MapBox?

See "OSM IDs" section in:
https://www.mapbox.com/developers/vector-tiles/mapbox-streets-v5/ or 
https://www.mapbox.com/developers/vector-tiles/mapbox-streets-v6/

**Comment by lukasmartinelli** [comment]
Yes because it seems trivial to implement into the `SELECT` clauses.

![screenshot_28_10_15_14_43](https://cloud.githubusercontent.com/assets/1288339/10790215/52b81e40-7d82-11e5-8bd3-8005e0856980.png)


But why are OSM ids even propagated to the vector layer?
Will cartographers search for an OSM id and style the single object differently?

**Comment by das-g** [comment]
> But why are OSM ids even propagated to the vector layer?

Maybe so stuff like 'Query Features' (question mark button on http://openstreetmap.org) can be implemented client-side?
